### ISSUE TITLE
Error begginer

### ISSUE THREAD

**Issue by ZaimSa** [angular/material#3595]
Hi guys,

I am trying to use material design and I have a problem which I don't understand. I just want to create an input for my formular, so I have this : 

    <md-input-container>
        <label style="color:gray;">Search</label>
        <input type="text" style="background-color:white">
    </md-input-container>

But this doesn't get the appearance it should have. I already add this : 

    
      <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular-cookies.js"></script>
      <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular-animate.min.js"></script>
      <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular-aria.min.js"></script>
      
      <script src="https://ajax.googleapis.com/ajax/libs/angular_material/0.8.3/angular-material.min.js">    
    </script>

If someone can help me please :)

**Comment by csvan** [comment]
Please use the Angular Material forum for general questions:

https://groups.google.com/forum/#!forum/ngmaterial

**Comment by Splaktar** [comment]
You'll also need the css included in your `<head>`. Take a look here for more info: https://material.angularjs.org/latest/#/getting-started

Also you might want to try the latest version, 0.10.0. And as stated, general questions should be asked on the forums or stackoverflow.

**Comment by ZaimSa** [comment]
Sorry I did'nt know it, 
Thank you :)
